airflow/models/dag.py

```diff
@@ -1840,6 +1840,7 @@ def set_task_instance_state(
         past: bool = False,
         commit: bool = True,
         session=NEW_SESSION,
+        group_id: str | None = None,
```

Comment:
@tquazi It would be nice if we can add new param to docstring :)

Reply:
Added group_id to the docstring.